CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 - Catalog: Connector filter is no longer split into ID and endpoint
```

Comment:
![grafik](https://github.com/user-attachments/assets/c7232d9d-5e50-4b2e-b3a9-7ce094e5863e)
"Use the Connector's JWKS URL to retrieve the connector certificate."